src/org/zaproxy/zap/extension/pscanrulesBeta/CSRFCountermeasures.java

```diff
@@ -293,4 +312,20 @@ public String getReference() {
     	return "Failed to load vulnerability reference from file";
     }
 
+    protected ExtensionAntiCSRF getExtensionAntiCSRF() {
+        return (ExtensionAntiCSRF) Control.getSingleton().getExtensionLoader().getExtension(ExtensionAntiCSRF.NAME);
+    }
+
+    protected String getCSRFIgnoreList() {
+        return Model.getSingleton().getOptionsParam().getConfig().getString("rules.csrf.ignorelist");
+    }
+
+    protected String getCSRFIgnoreAttName() {
+        return getConfig().getString("rules.csrf.ignore.attname", null);
```

Comment:
Shouldn't this also use the main configuration? (The rule configurations are not being injected into the passive scanners, that's done only for active ones?)